src/Conversion/ONNXToKrnl/ObjectDetection/NonMaxSuppression.cpp

```diff
@@ -423,7 +420,7 @@ struct ONNXNonMaxSuppressionOpLowering : public ConversionPattern {
                           createMath, selectedBox, otherBox, centerPointBox);
 
                       // Only proceed if IOU > iou_threshold.
```

Comment:
nit, does the comment needs to be update to `>=` ?

Reply:
Yes, just updated it. Thanks!